splinterd/tests/framework/circuit_builder/mod.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Framework of builders for integration testing
+
+mod builder;
+mod error;
+mod veil;
+
+pub use self::builder::{CircuitBuilder, CircuitData, CircuitService};
+pub use self::error::{AddScabbardServiceError, CircuitBuildError};
+pub use self::veil::scabbard::ScabbardCircuitBuilderVeil;
+pub use splinter::admin::messages::CreateCircuit;
```

Comment:
This should not be re-exported here.

Reply:
That was a leftover from when CreateCircuit was the result of the build() function, but I changed the return type to a new struct after talking it over with Peter. So yeah, this is definitely not necessary.